spec/models/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject do
+    User.create(name: 'userA', email: 'test1@yahoo.com', password: 'password')
+  end
+  before { subject.save }
+
+  it 'name should not be nil' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'should valid the name' do
```

Comment:
Thank you very much @Munsa1, i think the reason why you cannot see the styles is that you might not have bootstrap installed.

Reply:
- We also realised that so far everyone who has done the testing has  removed the confirmable and had an approval so we are not sure weather we should  remove it again since the  first code reviewer told us to add it back.  Please kindly assist.